go/exec/status.go

```diff
@@ -18,9 +18,46 @@
 
 package exec
 
+import (
+	"github.com/vitessio/arewefastyet/go/storage"
+)
+
 const (
 	StatusCreated  = "created"
 	StatusStarted  = "started"
 	StatusFailed   = "failed"
 	StatusFinished = "finished"
 )
+
+type benchmarkStats struct {
+	Benchmark_total    int
+	Benchmark_finished int
+	Benchmark_30Days   int
+}
+
+func GetBenchmarkStats(client storage.SQLClient) (benchmarkStats, error) {
```

Comment:
There is a mix of tabs and spaces indentation in this function, let's stick to tabs instead of mixing both